Python/pywarpx/WarpX.py

```diff
@@ -101,7 +103,16 @@ def write_inputs(self, filename='inputs', **kw):
 
         with open(filename, 'w') as ff:
 
+            prefix_old = ''
             for arg in argv:
+                # This prints the name of the input group (prefix) as a header
+                # before each group to make the input file more human readable
+                prefix_new = re.split(' |\.', arg)[0]
+                if prefix_new != prefix_old:
+                    if prefix_old != '':
+                        ff.write('\n')
+                    ff.write(f'# {prefix_new}\n')
+                    prefix_old = prefix_new
                 ff.write(f'{arg}\n')
```

Comment:
Slight readability update
```suggestion

                ff.write(f'{arg}\n')
```